src/main/java/Duke.java

```diff
@@ -1,10 +1,181 @@
+import java.util.Scanner;
 public class Duke {
+    public static int MAX_TASK = 100;
+    public static String logo = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+    public static String PROMETHEES =    " ____  ____  ____  _      _____ _____  _     _____  _____  ____\n"
+                                        +"/  __\\/  __\\/  _ \\/ \\__/|/  __//__ __\\/ \\ /|/  __/ /  __/ / ___\\\n"
+                                        +"|  \\/||  \\/|| / \\|| |\\/|||  \\    / \\  | |_|||  \\   |  \\   |    \\\n"
+                                        +"|  __/|    /| \\_/|| |  |||  /_   | |  | | |||  /_  |  /_  \\___ |\n"
+                                        +"\\_/   \\_/ \\_\\____/\\_/  \\|\\____\\  \\_/  \\_/ \\|\\____\\ \\____\\ \\____/\n";
+    public static String HORIZONTAL = "--------------------------------------";
+    public static String HELLO_GREET =
+            " My Son, I am Promethees, who had devoted his liver to liberate humanity from the Olympian's oppression\n"
+                    +" Tell me, what can I do for you?";
+    public static String GOOD_BYE = " Mission accomplished!";
+    public static String INSTRUCTION = "Invalid Command! Available Commands: bye, list, todo, deadline, event, done.";
+    public static int TODO_INDEX = 5;
+    public static int DEADLINE_INDEX = 9;
+    public static int EVENT_INDEX = 6;
+
+    public static void printWelcomeGreet() {
+        System.out.println(PROMETHEES);
+        System.out.println(HORIZONTAL);
+        System.out.println(HELLO_GREET);
+        System.out.println(HORIZONTAL);
+    }
+    public static void printGoodbye() {
+        System.out.println("\t" +HORIZONTAL);
+        System.out.println("\t"+GOOD_BYE);
+        System.out.println("\t" +HORIZONTAL);
+    }
+    public static void parseInput(int countTasks, Task[] tasks) {
+
+        //Input and read inputs from the user
+        Scanner in = new Scanner(System.in);
+        String line = in.nextLine();
+        String[] words = line.split(" ");
+        String command = words[0];
+
+        while (true) {
+            //System.out.println(command);
+            if (command.equalsIgnoreCase("bye")) {
+                return;
+            } else if (command.equalsIgnoreCase("list")) {
+                System.out.println(HORIZONTAL);
+                printTasks(countTasks, tasks);
+                System.out.println(HORIZONTAL);
+            } else if (command.equalsIgnoreCase("todo")) {
+                String newTodo = line.substring(TODO_INDEX);
+                countTasks = addTodos(countTasks, tasks, newTodo);
+                System.out.println("\tGot it. I've added this task: ");
+                System.out.println("\t"+tasks[countTasks - 1].getType() + tasks[countTasks - 1].getStatusIcon() + " " + tasks[countTasks - 1].getDescription());
```

Comment:
This line is repeated quite a few times. Can you try to refactor into a new function?

Reply:
I will move it to addTodos, addEvents and addDeadlines functions